docs/sources/configure/storage.md

```diff
@@ -252,6 +257,12 @@ schema_config:
         period: 24h
 ```
 
+`service_account` should contain JSON from either a GCP Console client_credentials.json file or a GCP service account key. If this value is blank, most services will fall back to GCP's Application Default Credentials strategy. See [How Application Default Credentials works](https://cloud.google.com/docs/authentication/application-default-credentials) for more information about ADC.
+
+{{< admonition type="note" >}}
+NOTE: GCP recommends [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) instead of a service account key.
```

Comment:
You don't need the word Note here, the admonition shortcode already has it in line 262.
```suggestion
GCP recommends [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) instead of a service account key.
```